src/components/layout/CommentInput.jsx

```diff
@@ -40,14 +49,16 @@ const CommentInput = () => {
             required
             onFocus={() => setFocus(true)}
             onChange={(event) => setInput(event.target.value)}
+            onKeyPress={submitCommentHandler}
           />
           {focus ? (
             <div className='comment-button-container'>
               <button
+                onClick={() => submitComment()}
                 className='submit-btn '
                 data-cy='comment-btn'
                 name='comment'
-                type='submit'>
+                type='button'>
                 Submit
```

Comment:
Do you need to specify type button on a button element?

Reply:
Hmm we didn't try to not use any type, but the ` type='submit' ` fucked us hard for a while. it updated the site when we clicked the button but not enter and we couldn't figure it out why for a while then Christer changed the type to button and it worked so we decided not to touch that again for a while 😅